### PR TITLE
Update Spanish Translations

### LINKED ISSUE #404: Add social discussion capabilities via Microsoft Embedded Social
I've been working with some folks from Microsoft Research in prototyping social networking/discussion features in OBA via a new platform called ~~SocialPlus~~ Embedded Social.  The main benefit to SocialPlus is that its an SDK that allows social actions within the context of the app itself, as opposed to integration with other social platforms that kick you out into their own application.  It does this by exposing most of the content via a REST API.

We've currently discussed adding social features to OBA surrounding bus stops and routes.

Here's a YouTube video showing a demo of the current integration that Microsoft has been working on - it's still a work in progress:
https://youtu.be/XqEUD61ZIvs

The Google Drawings document for our mockups is here:
https://docs.google.com/drawings/d/1X7EB18ENI1EqgAP-Xqcf0LeL6ywb8SxbMdlPH0YfyIE/edit?usp=sharing

There are Powerpoint slides in this oba-dev group post detailing the effort in greater detail as well:
https://groups.google.com/d/msg/onebusaway-developers/Dl93acfEtsw/hP7lYsqKBwAJ

### LINKED ISSUE #816: Can't create shortcuts on Android 8.0 Oreo
**Summary:** 

If you try to create a starred stop shortcut on an Android 8.0 Oreo device, OBA Android says that the shortcut was created but no shortcut icon appears.

This also applies to other app shortcuts too (Night light, etc.) - anything that uses `Intent.setAction(INSTALL_SHORTCUT)`.

Looks like Google changed the behavior on creating these shortcuts in Android 8.0 so this now fails silently.

https://developer.android.com/about/versions/oreo/android-8.0-changes.html#as says:

>The com.android.launcher.action.INSTALL_SHORTCUT broadcast no longer has any effect on your app, because it is now a private, implicit broadcast. Instead, you should create an app shortcut by using the [requestPinShortcut()](https://developer.android.com/reference/android/content/pm/ShortcutManager.html#requestPinShortcut(android.content.pm.ShortcutInfo, android.content.IntentSender)) method from the [ShortcutManager](https://developer.android.com/reference/android/content/pm/ShortcutManager.html) class.

See https://stackoverflow.com/questions/44744184/how-to-create-shortcuts-on-android-o-when-targetting-less-than-it for details.

**Steps to reproduce:** 

1. Create starred stop
1. Go to Starred Stops fragment
1. Long-press on stop and tap "Create shortcut on home screen"

**Expected behavior:** 

A shortcut to that stop should appear on the home screen

**Observed behavior:** 

Stop shortcut doesn't appear on home screen

**Device and Android version:** 

Samsung Galaxy S8+ w/ Android 8.0 (Samsung beta) and Nexus 5X w/ Android 8.0 (production release)

### PR DESCRIPTION
**Pull request by themonki**
Update Spanish Translations

* Based on commit 3bd995d - Fix #816 - Fix shortcuts on Android 8.0 Oreo (#820)
* Based on commit 82feedb - add a contributor
* Based on commit fd6ff09 - Fix #404 - Add social functionality via Embedded Social (#744)
* Based on commit c6422a3 - Update "What's New" to prepare for release
* Based on commit fd48a27 - Update "What's New" to include bug fixes

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [ ] Run the unit tests with `gradlew connectedObaGoogleDebugAndroidTest connectedObaAmazonDebugAndroidTest` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: onebusaway-android/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1096,13 +1096,13 @@
...
 
     <!-- Social -->
-    <string name="embedded_social_restriction">Enable social features</string>
-    <string name="es_missing_stop_conversation">Social discussion not yet available for this stop</string>
-    <string name="es_missing_route_conversation">Social discussion not yet available for this route</string>
-    <string name="es_msg_general_create_user_success">Created Embedded Social account</string>
-    <string name="es_msg_general_signin_success">Signed into Embedded Social</string>
-    <string name="es_msg_general_signout_success">Signed out of Embedded Social</string>
-    <string name="es_msg_general_account_deleted">Embedded Social account deleted</string>
+    <string name="embedded_social_restriction">Habilitar funciones sociales</string>
```
Comment (barbeau) [inline onebusaway-android/src/main/res/values/strings.xml]: I believe you switched the Spanish and English versions in this and `-es/strings.xml`.  Could you swap them?

**Comment by barbeau** [comment]
Thanks @themonki!  I went ahead and made the change and merged the new translations so they make the next release.  Thanks again for keeping these update to date!

**Comment by themonki** [comment]
Sorry! :(

**Comment by barbeau** [comment]
No worries!  I just appreciate the help with the translations.

**Pull request merged**